client/components/Omnichannel/modals/ForwardChatModal.js

```diff
@@ -0,0 +1,93 @@
+import React, { useCallback, useEffect, useRef, useState } from 'react';
+import { Field, Button, TextAreaInput, Icon, ButtonGroup, Modal, Box } from '@rocket.chat/fuselage';
+import { useMutableCallback } from '@rocket.chat/fuselage-hooks';
+
+import { useTranslation } from '../../../contexts/TranslationContext';
+import { useForm } from '../../../hooks/useForm';
+import ModalSeparator from '../../ModalSeparator';
+import DepartmentAutoComplete from '../../../views/omnichannel/DepartmentAutoComplete';
+import { UserAutoComplete } from '../../AutoComplete';
+import { useEndpoint } from '../../../contexts/ServerContext';
+
+const ForwardChatModal = ({ onForward, onCancel, ...props }) => {
+	const t = useTranslation();
+
+	const ref = useRef();
+
+	useEffect(() => {
+		if (typeof ref?.current?.focus === 'function') {
+			ref.current.focus();
+		}
+	}, [ref]);
+
+	const { values, handlers } = useForm({ departmentName: '', username: '', comment: '' });
+	const { departmentName, username, comment } = values;
+	const [userId, setUserId] = useState('');
+
+	const { handleDepartmentName, handleUsername, handleComment } = handlers;
+	const userInfo = useEndpoint('GET', `users.info?username=${ username }`);
+
+
+	const handleSend = useCallback(() => {
```

Comment:
since you are using useMutableCallback for the other handlers, I think you can use the same here.